README.md

```diff
@@ -33,18 +33,119 @@ We currently support [ONNX runtime](https://github.com/microsoft/onnxruntime) al
+If you'd like to play with the examples or need the bleeding edge of the code and can't wait for a new release, you can install the library from source as follows:
+
+```bash
+python -m pip install git+https://github.com/huggingface/optimum.git
+```
+
+## Quickstart
+
+At its core, 🤗 Optimum uses _configuration objects_ to define parameters for optimization on different accelerators. These objects are then used to instantiate dedicated _optimizers_, _quantizers_, and _pruners_`. For example, here's how you can apply dynamic quantization with ONNX Runtime:
+
+```python
+from optimum.onnxruntime import ORTConfig, ORTQuantizer
+
+# The model we wish to quantize
+model_ckpt = "distilbert-base-uncased-finetuned-sst-2-english"
+# The type of quantization to apply
+ort_config = ORTConfig(quantization_approach="dynamic")
+quantizer = ORTQuantizer(ort_config)
+# Quantize the model!
+quantizer.fit(model_ckpt, output_dir="onnx", feature="sequence-classification")
+```
+
+In this example, we've quantized a model from the Hugging Face Hub, but it could also be a path to a local model directory. The `feature` argument in the `fit()` method corresponds to the type of task that we wish to quantize the model for. The result from applying the `fit()` method is a `model-quantized.onnx` file that can be used to run inference. Here's an example of how to load an ONNX Runtime model and generate predictions with it:
+
+```python
+from datasets import Dataset
+from transformers import AutoTokenizer
+from optimum.onnxruntime import ORTModel
+
+# Load quantized model
+ort_model = ORTModel("model-quantized.onnx", quantizer.onnx_config)
+# Create a dataset or load one from the Hub
+ds = Dataset.from_dict({"sentence": ["I love burritos!"]})
+# Tokenize the inputs & convert to PyTorch tensors
+tokenizer = AutoTokenizer.from_pretrained(model_ckpt)
+
+def preprocess_fn(ex):
+    return tokenizer(ex["sentence"])
+
+tokenized_ds = ds.map(preprocess_fn, remove_columns=ds.column_names)
+tokenized_ds.set_format("torch")
+# Create dataloader and run evaluation
+dataloader = DataLoader(tokenized_ds)
+ort_outputs = ort_model.evaluation_loop(dataloader)
+# Extract logits!
+ort_outputs.predictions
+```
+
+Similarly, you can apply static quantization by simply changing the `quantization_approach` in the `ORTConfig` object:
+
+```python
+ort_config = ORTConfig(quantization_approach="dynamic")
+```
+
+Static quantization relies on feeding batches of data through the model to identify which activations should be quantized. To support this, 🤗 Optimum allows you to provide a _calibration dataset_. The calibration dataset can be a simple `Dataset` object from the 🤗 Datasets library, or any dataset that's hosted on the Hugging Face Hub. For this example, we'll pick the [`sst2`](https://huggingface.co/datasets/glue/viewer/sst2/test) dataset that the model was originally trained on:
```

Comment:
I think "to identify which activations should be quantized" should be replaced by something like "to estimate the activations quantization parameters"